bindings/runner/src/common.rs

```diff
@@ -593,3 +623,86 @@ fn get_cell_id_for_role_name(app_info: &AppInfo, role_name: &RoleName) -> anyhow
         _ => anyhow::bail!("Cell not provisioned"),
     }
 }
+
+/// Runs an instance of the Holochain conductor, using the configuration built from the
+/// [`HolochainAgentContext::holochain_config`] and stores the running process in
+/// [`HolochainAgentContext::holochain_runner`].
+pub fn run_holochain_conductor<SV: UserValuesConstraint>(
+    ctx: &mut AgentContext<HolochainRunnerContext, HolochainAgentContext<SV>>,
+) -> WindTunnelResult<()> {
+    if ctx.runner_context().get_connection_string().is_some() {
+        log::info!(
+            "connection-string is set so assuming a Holochain conductor is running externally"
+        );
+        return Ok(());
+    }
+
+    if let Ok(holochain_path) = env::var("WT_HOLOCHAIN_PATH") {
+        if holochain_path.is_empty() {
+            bail!("'WT_HOLOCHAIN_PATH' set to empty string");
+        }
+        if holochain_path == "holochain" {
+            log::warn!("'WT_HOLOCHAIN_PATH' set to 'holochain' so looking in user's 'PATH'");
```

Comment:
Could remove this warning, but it's to make sure people know that it won't use the bin in the local path. Could also change it to:


```suggestion
            log::warn!("'WT_HOLOCHAIN_PATH' set to 'holochain' so looking in user's 'PATH', use 'WT_HOLOCHAIN_PATH=./holochain' to use a binary in local path");
```